src/sinks/prometheus/remote_write.rs

```diff
@@ -381,6 +390,10 @@ mod integration_tests {
                 ca_file: Some(tls::TEST_PEM_CA_PATH.into()),
                 ..Default::default()
             }),
+            auth: Some(Auth::Basic {
+                user: String::from("someuser"),
+                password: String::from("somepassword"),
+            }),
```

Comment:
Since our InfluxDB configuration doesn't require a name and password, this is more of a configuration test, so it should probably go into the non-integration tests.